README.md

```diff
@@ -42,12 +37,20 @@ NOTE: RPM, DEB, and Arch packages are planned for future releases.
 
 ### Running puppet_webhook
 
-Once installed, you can run the application by simply executing the `puppet_webhook` binary.
+#### Quick Start
+Simply run `# puppet_webhook` after installation to start puppet_webhook in non-daemon mode on your system. This is great for testing the server out.
 
-This binary will default to using the bundled configuration files and run in a non-daemon mode. This mode is useful for debugging purposes, but it probably not ideal for production use.
-You can also set the `server_type` option in `/etc/puppet_webhook/server.yml` to `daemon` to run the application in the background. By default the application will log to `/var/log/puppet_webhook/access.log`.
+#### CLI Tool
+The `puppet_webhook` CLI command has several options you can pass it as well.
 
-NOTE: During the Prerelease stage, the `/etc/puppet_webhook` and `/var/log/puppet_webhook` directories need to be manually created. This will be fixed in for General Availability.
+To see these options run `# puppet_webook -h` in your terminal to see all the options.
+
+#### Start using Rack-based server software
+A `config.ru` file is also packaged with the application to provide users with the ability to start the app using their own rack-based server such as
```

Comment:
If we want valid markdown we need a newline between heading and text (in general nobody cares)